packages/ua-blocker/src/generated.ts

```diff
@@ -76,3 +76,5 @@ Disallow: /
+export const ALL_BOTS_REGEX = /(\x41I2BOT|\x41I2BOT\x2dDOLMA|\x41IHITBOT|\x41MAZONBOT|\x41NDIBOT|\x41NTHROPIC\x2dAI|\x41PPLEBOT|\x41PPLEBOT\x2dEXTENDED|\x42EDROCKBOT|\x42RIGHTBOT\x201\.0|\x42YTESPIDER|\x43CBOT|\x43HATGPT\x2dUSER|\x43LAUDE\x2dSEARCHBOT|\x43LAUDE\x2dUSER|\x43LAUDE\x2dWEB|\x43LAUDEBOT|\x43OHERE\x2dAI|\x43OHERE\x2dTRAINING\x2dDATA\x2dCRAWLER|\x43OTOYOGI|\x43RAWLSPACE|\x44IFFBOT|\x44UCKASSISTBOT|\x45CHOBOXBOT|\x46ACEBOOKBOT|\x46ACTSET_SPYDERBOT|\x46IRECRAWLAGENT|\x46RIENDLYCRAWLER|\x47OOGLE\x2dCLOUDVERTEXBOT|\x47OOGLE\x2dEXTENDED|\x47OOGLEOTHER|\x47OOGLEOTHER\x2dIMAGE|\x47OOGLEOTHER\x2dVIDEO|\x47PTBOT|\x49ASKSPIDER\/2\.0|\x49CC\x2dCRAWLER|\x49MAGESIFTBOT|\x49MG2DATASET|\x49SSCYBERRISKCRAWLER|\x4bANGAROO\x20BOT|\x4dETA\x2dEXTERNALAGENT|\x4dETA\x2dEXTERNALAGENT|\x4dETA\x2dEXTERNALFETCHER|\x4dETA\x2dEXTERNALFETCHER|\x4dISTRALAI\x2dUSER\/1\.0|\x4eOVAACT|\x4fAI\x2dSEARCHBOT|\x4fMGILI|\x4fMGILIBOT|\x4fPERATOR|\x50ANGUBOT|\x50ANSCIENT|\x50ANSCIENT\.COM|\x50ERPLEXITY\x2dUSER|\x50ERPLEXITYBOT|\x50ETALBOT|\x50HINDBOT|\x51UALIFIEDBOT|\x51UILLBOT|\x51UILLBOT\.COM|\x53BINTUITIONSBOT|\x53CRAPY|\x53EMRUSHBOT\x2dOCOB|\x53EMRUSHBOT\x2dSWA|\x53IDETRADE\x20INDEXER\x20BOT|\x54IKTOKSPIDER|\x54IMPIBOT|\x56ELENPUBLICWEBCRAWLER|\x57EBZIO\x2dEXTENDED|\x57PBOT|\x59ANDEXADDITIONAL|\x59ANDEXADDITIONALBOT|\x59OUBOT)/;
+export const NON_RESPECTING_BOTS_REGEX = /(\x41NDIBOT|\x41NTHROPIC\x2dAI|\x41PPLEBOT|\x42EDROCKBOT|\x42RIGHTBOT\x201\.0|\x42YTESPIDER|\x43CBOT|\x43LAUDE\x2dSEARCHBOT|\x43LAUDE\x2dUSER|\x43LAUDE\x2dWEB|\x43LAUDEBOT|\x43OHERE\x2dAI|\x43OHERE\x2dTRAINING\x2dDATA\x2dCRAWLER|\x43RAWLSPACE|\x44IFFBOT|\x44UCKASSISTBOT|\x45CHOBOXBOT|\x46ACEBOOKBOT|\x46ACTSET_SPYDERBOT|\x46RIENDLYCRAWLER|\x47OOGLE\x2dCLOUDVERTEXBOT|\x47OOGLE\x2dEXTENDED|\x47OOGLEOTHER|\x47OOGLEOTHER\x2dIMAGE|\x47OOGLEOTHER\x2dVIDEO|\x49ASKSPIDER\/2\.0|\x49MAGESIFTBOT|\x49MG2DATASET|\x49SSCYBERRISKCRAWLER|\x4bANGAROO\x20BOT|\x4dETA\x2dEXTERNALAGENT|\x4dETA\x2dEXTERNALFETCHER|\x4dETA\x2dEXTERNALFETCHER|\x4eOVAACT|\x4fAI\x2dSEARCHBOT|\x4fMGILI|\x4fMGILIBOT|\x4fPERATOR|\x50ANGUBOT|\x50ANSCIENT|\x50ANSCIENT\.COM|\x50ERPLEXITY\x2dUSER|\x50ERPLEXITYBOT|\x50HINDBOT|\x51UALIFIEDBOT|\x51UILLBOT|\x51UILLBOT\.COM|\x53BINTUITIONSBOT|\x53CRAPY|\x53EMRUSHBOT\x2dOCOB|\x53EMRUSHBOT\x2dSWA|\x53IDETRADE\x20INDEXER\x20BOT|\x54IKTOKSPIDER|\x54IMPIBOT|\x56ELENPUBLICWEBCRAWLER|\x57EBZIO\x2dEXTENDED|\x57PBOT|\x59ANDEXADDITIONAL|\x59ANDEXADDITIONALBOT|\x59OUBOT)/;
```

Comment:
I included the regular expressions in the generated output. This allows them to be compiled when the expressions are evaluated, rather than at runtime.

I'd recommend removing the string literal arrays from code generation, but I'll leave you to make that decision